Sources/LicensePlistCore/Entity/CocoaPods.swift

```diff
@@ -6,12 +6,21 @@ public struct CocoaPods: Library {
     public let name: String
     public let nameSpecified: String?
     public let version: String?
+    public let source: String?
+    
+    public init(name: String, nameSpecified: String?, version: String?) {
+        self.name = name
+        self.nameSpecified = nameSpecified
+        self.version = version
+        self.source = "https://cocoapods.org/pods/\(name)"
+    }
```

Comment:
@divadretlaw 

How about this?

```suggestion
    public var source: String? { "https://cocoapods.org/pods/\(name)" }

    public init(name: String, nameSpecified: String?, version: String?) {
        self.name = name
        self.nameSpecified = nameSpecified
        self.version = version
    }
```

Reply:
Seems good, then we can drop the custom initializer.